composer.json

```diff
@@ -92,9 +92,6 @@
                 "Undefined property: stdClass::$region in field_group_form_process().": "https://www.drupal.org/files/issues/2020-06-15/3059614-37.patch",
                 "Remove dependency on jQuery UI Accordion for the accordion field group formatter": "https://www.drupal.org/files/issues/2021-10-16/3154304-2.patch"
             },
-            "drupal/data_policy": {
-                "9.2.x compatibility": "https://git.drupalcode.org/project/data_policy/-/merge_requests/12.diff"
```

Comment:
@tbsiqueira I would recommend adding the patch as the file from the issue instead of MR.

Reply:
@Ressinel This change is already available in the new version.

Reply:
@navneet0693 What new version do you mean?

Reply:
@Ressinel, we released a 2.0.0-beta1 version for data_policy containing the latest needed changes, this is why we removed the patch and changes the module version on composer.json

Reply:
@tbsiqueira  check, thanks!